src/themes/creator.js

```diff
@@ -9,696 +9,45 @@ LICENSE file in the root directory of this source tree.
 import type {ThemeT} from '../styles/types.js';
```

Comment:
I don't see createTheme being used anywhere else in this diff. Is it only here for backward compatibility?

Reply:
For backward compatibility as it's part of the current API. But there is still value in it in term of a custom theme creation, the only thing would need to be changed in the future is the structure of the primitive.